app/tests/e2e/issuer/did_key_bbs/test_ld_bbs.py

```diff
@@ -106,6 +107,7 @@ async def test_send_jsonld_key_bbs(
     )
 
     # Check if Alice received the credential
+    asyncio.sleep(0.1)  # credential may take moment to reflect after webhook
```

Comment:
It would've been nice if I added `await` in front of the coroutine, but it's the thought that counts :see_no_evil: